src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -88,6 +88,7 @@ public class DurableTaskExtension :
         private bool isTaskHubWorkerStarted;
         private HttpClient durableHttpClient;
         private EventSourceListener eventSourceListener;
+        private Guid extensionGUID;

```

Comment:
Can this be made `readyonly`? It looks like we only assign it in the constructor. Also, per standard C# naming conventions, it should be `extensionGuid`.
```suggestion
        private readonly Guid extensionGuid;
```

Reply:
Sure, good idea. I'll incorporate this feedback without the GitHub UI so that I can easily propagate the naming change via VS